core/types/transaction_signing.go

```diff
@@ -100,11 +100,16 @@ type Signer interface {
 // `eth_sign` based signature hash.
 type OVMSigner struct {
 	EIP155Signer
```

Comment:
Should probably refactor out `EIP155Signer` and add in the last method needed to implement the `Signer` interface and not rely on the `EIP155Signer`'s method as fallback